src/quo2/foundations/colors.cljs

```diff
@@ -272,24 +272,32 @@
      ([color suffix]
       (custom-color color suffix nil))
      ([color suffix opacity]
-      (let [hex?  (not (keyword? color))
-            color (if (hex-string? (get colors-map color))
-                    (get colors-map color)
-                    (get-in colors-map [color suffix]))]
+      (let [hex?           (not (keyword? color))
+            resolved-color (cond hex?                                 color
+                                 (hex-string? (get colors-map color)) (get colors-map color)
+                                 :else                                (get-in colors-map
+                                                                              [color suffix]))]
         (cond
-          (and opacity hex?) (alpha color (/ opacity 100))
-          opacity            (alpha color (/ opacity 100))
-          hex?               color
-          :else              color))))))
+          (and opacity hex?) (alpha resolved-color (/ opacity 100))
+          opacity            (alpha resolved-color (/ opacity 100))
+          hex?               resolved-color
```

Comment:
question : is this condition necessary ? `hex? resolved-color`
I think you already take care of that in the let binding above ?

Reply:
yeah good point, I think it should be removed 👍

Reply:
thanks for spotting that @siddarthkay, also found other redundancies. was straightforward to refactor with tests in place :)

Reply:
Awesome!!!